helm/fiftyone-teams-app/README.md

```diff
@@ -416,6 +416,7 @@ appSettings:
 | apiSettings.image.pullPolicy | string | `"Always"` | Instruct when the kubelet should pull (download) the specified image. One of `IfNotPresent`, `Always` or `Never`. [Reference][image-pull-policy]. |
 | apiSettings.image.repository | string | `"voxel51/fiftyone-teams-api"` | Container image for the teams-api. |
 | apiSettings.image.tag | string | `""` | Image tag for teams-api. Defaults to the chart version. |
+| apiSettings.labels | object | `{}` |  |
```

Comment:
When the `values.yaml` comments start with `# -- ` mkdocs will take that description and place it here in the table.